code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -974,7 +974,7 @@
 									if(head_organ.take_damage(30, 0))
 										H.UpdateDamageIcon(1)
 									head_organ.disfigure("burn")
-									H.emote("scream",,, 1)
+									H.emote("scream", TRUE)
```

Comment:
If I'm reading this correctly the second parameter to `emote` is m_type, which ends up becoming the third parameter to `run_emote`: `type_override` which is probably not what you intended? And why would an unconscious vampire be able to scream anyway?